spec/requests/schoolings_controller_spec.rb

```diff
@@ -18,4 +34,43 @@
              params: { confirmed_director: "1" }
     end
   end
+
+  describe "retry_eligibile_payment_requests" do
+    before do
+      schooling.update!(end_date: Date.parse("2024-06-21"))
+    end
+
+    context "when the payment request is retry eligible" do
+      it "Does not return abrogated decision error" do
+        delete abrogate_decision_school_year_class_schooling_path(schooling.classe.school_year,
+                                                                  class_id: schooling.classe.id, id: schooling.id),
+               params: { confirmed_director: "1" }
+        expect(payment_request.last_transition.metadata).not_to include(error_message)
+      end
+    end
+
+    context "when the payment request is retry eligible" do
```

Comment:
T'as deux fois le même context "when the payment request is retry eligible"

Reply:
Rubocop l'a vu, j'ai fix